opencti-platform/opencti-graphql/src/domain/stream.js

```diff
@@ -58,15 +59,29 @@ export const createGroupRelation = async (context, user, collectionId, groupId)
   return findById(context, user, collectionId);
 };
 export const findAll = (context, user, args) => {
-  return listEntities(context, user, [ENTITY_TYPE_STREAM_COLLECTION], args);
+  if (user) {
+    return listEntities(context, user, [ENTITY_TYPE_STREAM_COLLECTION], args);
+  }
+  return listEntities(
+    context,
+    SYSTEM_USER,
+    [ENTITY_TYPE_STREAM_COLLECTION],
+    {
+      ...(args ?? {}),
+      filters: [
+        ...(args.filters ?? []),
+        { key: ['public'], values: ['true'] },
```

Comment:
Forcing public to true in findAll api?

Reply:
In case we don't have a user specified yes. To be able to query the public page ?